README.md

```diff
@@ -47,80 +47,94 @@ ShapeShift's OSS 2nd generation Web application. (Under Development)
 
 ## Quick Start
 
-If you are using Linux and MacOS it works out of the box following the steps.<br/>
-⚠️ On Windows you should use the _Windows Subsystem for Linux_ (WSL).
+If you are using Linux and MacOS it works out of the box following these steps:
 
-- Clone the repo
+1. Clone the repo
 
-- (optional) Make sure you're using the right Node.js version.
+2. (optional) Make sure you're using the right Node.js version.
 
   ```sh
   nvm use
   ```
 
-- Install Dependencies:
+3. Install Dependencies:
 
   ```sh
   # This is short for `yarn install`; be sure to use `yarn install --frozen-lockfile` instead if you're setting up a CI pipeline or trying to duplicate a historical build.
   yarn
   ```
 
-- Run `yarn env dev` to generate a `.env` file
+4. Run `yarn env dev` to generate a `.env` file
 
-### Commands
+> ⚠️ On Windows, _[Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/)_ (WSL) is required but **WE DO NOT SUPPORT THIS ENVIRONMENT, SO USE IT AT YOUR OWN RISK**.
+>
+> If you do use WSL, make sure to store your clone of the repo on the WSL filesystem in order to avoid issues with line endings and to enjoy better performance.
+>
+> **Please do NOT ask our Engineering team for further help with WSL.**
 
-Runs the app in the development mode. Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### Commands
 
-The page will reload if you make edits. You will also see any lint errors
-in the console.
+To run the app in the development mode:
 
 ```sh
-    yarn dev
+yarn dev
 ```
 
-Launches the test runner in the interactive watch mode.<br /> See the section
-about
-[running tests](https://facebook.github.io/create-react-app/docs/running-tests)
-for more information.
-It also creates a html page you can interact with at the root level of the project in `/coverage`.
+> It opens [http://localhost:3000](http://localhost:3000) to view it in the browser and the page will reload if you make edits.
```

Comment:
Very nitpicky but `this` wordage instead of `it` (referring to the aforementioned command) seems more natural, but this is already an improvement as is

Reply:
Alternatively, ditching `it` and keeping the verb in present tense also works